src/controllers/route.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const path = require('path');
+const router = express.Router();
+
+const home = require('../views/home');
+
+router.get('/', (req, res) => {
+   res.render('home', { activePage: {home: true}});
+});
+
```

Comment:
Why are you sending data on both routes (activePage: {home: true}) ? Remember the second argument is where you send data for handlebars to use. This active page data was just used in the code along to set a class which underlined the link.

You can omit this for now, just use res.render('home');